assets/js/previews/single-product-block.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * External dependencies
+ */
+import { WC_BLOCKS_ASSET_URL } from '@woocommerce/block-settings';
+
+export const singleProductBlockPreview = (
+	<img
+		src={ WC_BLOCKS_ASSET_URL + 'img/single-product-preview.svg' }
+		alt="Single Product Preview"
```

Comment:
Should this be localized? I'm not sure if it's possible to navigate to this image with assistive technology, if not, I think it would be just ok to leave `alt` blank.

PD: I noticed the alt text for the `Checkout` and `Product grid` previews is not localizable either, so we might handle all of them in a follow-up if it's easier.